.github/workflows/tests_t8code_parallel.yml

```diff
@@ -270,4 +271,4 @@ jobs:
       uses: actions/upload-artifact@v3
       with:
         name: test-suite_release_MPI_cxx.log
-        path: build_release_cpp/test-suite.log
```

Comment:
Can you please add the line break at the end?

Reply:
Fixed in 3c580c97.